evil-textobj-tree-sitter.el

```diff
@@ -135,10 +135,10 @@ They will be order with captures with point inside them first then the
 ones that follow.  If a `QUERY' alist is provided, we make use of that
 instead of the builtin query set."
```

Comment:
I guess the doc could probably use a update to mention this behavior. Maybe even in `evil-textobj-tree-sitter-get-textobj`.

Reply:
done

Reply:
Looks like the CI is failing with the new doc update.

Reply:
fixed